pkg/operator/options/options.go

```diff
@@ -157,6 +161,16 @@ func (o *Options) Parse(fs *FlagSet, args ...string) error {
 	o.FeatureGates = gates
 	o.PreferencePolicy = PreferencePolicy(o.preferencePolicyRaw)
 	o.MinValuesPolicy = MinValuesPolicy(o.minValuesPolicyRaw)
+
+	// Parse comma-separated additional nodepool metric labels
+	if o.rawAdditionalNodePoolMetricLabels != "" {
+		o.AdditionalNodePoolMetricLabels = lo.Map(strings.Split(o.rawAdditionalNodePoolMetricLabels, ","), func(s string, _ int) string {
+			return strings.TrimSpace(s)
+		})
+	} else {
+		o.AdditionalNodePoolMetricLabels = []string{}
+	}
```

Comment:
nit: less code paths 🤷 
```suggestion
	o.AdditionalNodePoolMetricLabels = lo.Map(strings.Split(o.rawAdditionalNodePoolMetricLabels, ","), func(s string, _ int) string {
		return strings.TrimSpace(s)
	})
```